src/Avatar/Avatar.js

```diff
@@ -14,13 +15,18 @@ export const styleSheet = createStyleSheet('Avatar', (theme) => {
       justifyContent: 'center',
       width: 40,
       height: 40,
-      fontSize: 24,
+      fontSize: 20,
       borderRadius: '50%',
       overflow: 'hidden',
+      userSelect: 'none',
     },
     defaultColor: {
-      color: palette.getContrastText(palette.text.disabled),
-      background: palette.text.disabled,
+      color: palette.background.default,
+      backgroundColor: emphasize(palette.background.default, 0.26),
+    },
+    icon: {
```

Comment:
> That would allow us to avoid the cloneElement.

It would, but I had to put it back for childrenClassName. 

I agree the styles are redundant though - they're an artefact from Avatar on `master` where these were dynamic based on the `size` prop.